twilio-flex-webchat/README.md

```diff
@@ -0,0 +1,55 @@
+# Daily Demo: Twilio Flex Webchat integration
+
+This project demonstrates one way to integrate a [Daily Prebuilt](https://daily.co/prebuilt) call with [Twilio's Flex Webchat UI](https://www.twilio.com/docs/flex/developer/webchat/setup).
+
+The intention is to extend Twilio's web chat widget by adding the option to add a video call with the chat's moderator.
+
+<img src="daily_twilio_demo.png" alt="Daily call with Flex WebChat UI">
+
+## Running this demo locally
+
+In the project directory, you can run:
+
+```bash
+yarn
+yarn start
+```
+
+Open http://localhost:3000 to view it in the browser.
+
+## Setting your local environment variables
+
+Before starting your server, create an `.env` file with the following values:
+
+```
+REACT_APP_TWILIO_ACCOUNT_SID="your-account-sid"
+REACT_APP_TWILIO_FLEX_FLOW_SID="your-flex-flow-sid"
+REACT_APP_DAILY_API_KEY="your-daily-api-key"
+```
+
+You can also copy the `.env.sample` file provided in the root directory.
+
+To find these values, follow the directions below.
+
+---
+
+## Setting up Daily and Twilio accounts
+
+To use this demo, you will need a Daily and a Twilio account. Both offer free account options.
+
+### Getting set up with Daily
+
+To start, [create a Daily account](https://dashboard.daily.co/signup).
+
+Once you have an account and are logged into the [Daily Dashboard](https://dashboard.daily.co), you can find your API key on the [Developers page](https://dashboard.daily.co/developers).
+
+### Getting set up with Twilio
+
+Start by setting up a [Twilio account](https://www.twilio.com/try-twilio).
```

Comment:
I've got this running and it looks great! 🥳 

The only thing that I got hung up on was where to find the things I needed for Twilio. I have a couple of Twilio projects, and each has its own SID. So I wondered, "Do I need to create a new project?" I didn't know, so I just thought, "I'll go find Flex in the Twilio dashboard." When I did that and created a new Flex project, I was able to see both the SID for the Twilio account and for Flex Flow in: https://flex.twilio.com/admin/developers

I wouldn't necessarily change anything in the instructions, because I doubt my Twilio dashboard situation will be the common one, but I thought I'd share in case useful.